main.go

```diff
@@ -262,8 +267,12 @@ func main() {
 	flag.StringVar(&username, "username", "", "cql username for authentication")
 	flag.StringVar(&password, "password", "", "cql password for authentication")
 
-	flag.BoolVar(&tls, "tls", false, "use TLS encryption")
-
+	flag.BoolVar(&tlsEncryption, "tls", false, "use TLS encryption")
+	flag.StringVar(&serverName, "tls-server-name", "", "TLS server hostname")
+	flag.BoolVar(&hostVerification, "tls-host-verification", false, "verify server certificate")
+	flag.StringVar(&caCertFile, "tls-ca-cert-file", "", "path to CA certificate file, needed to enable encryption")
+	flag.StringVar(&clientCertFile, "tls-client-cert-file", "", "path to client certificate file, needed to enable client certificate authentication")
+	flag.StringVar(&clientKeyFile, "tls-client-key-file", "", "path to client key file, needed to enable client certificate authentication")
 	flag.StringVar(&hostSelectionPolicy, "host-selection-policy", "token-aware", "set the driver host selection policy (round-robin,token-aware,dc-aware),default 'token-aware'")
```

Comment:
Nit: add a blank line before the line with the `hostSelectionPolicy` parameter. The `hostSelectionPolicy` parameter is unrelated from TLS options, so it makes sense to separate it visually.

Reply:
done